integreat_cms/static/src/js/menu.ts

```diff
@@ -2,7 +2,9 @@ document.addEventListener("DOMContentLoaded", () => {
     const mobile = document.querySelector("#mobile-menu");
     const sidebar = document.querySelector("#primary-navigation");
 
-    mobile.addEventListener("click", () => {
-        sidebar.classList.toggle("-translate-x-full");
-    });
+    if (mobile) {
+        mobile.addEventListener("click", () => {
+            sidebar.classList.toggle("-translate-x-full");
+        });
+    }
```

Comment:
```suggestion
    mobile?.addEventListener("click", () => {
        sidebar.classList.toggle("-translate-x-full");
    });
```

I find the [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html) syntax more concise.

Reply:
Thanks